x-pack/platform/plugins/private/reporting/public/management/components/scheduled_report_flyout_content.tsx

```diff
@@ -0,0 +1,407 @@
+    throw new Error('Cannot schedule an export without an objectType or sharingData');
+  }
+  const {
+    application: { capabilities },
+    http,
+    actions: { validateEmailAddresses },
+    notifications: { toasts },
+    userProfile: userProfileService,
+  } = useKibana().services;
+  const { data: userProfile, isLoading: isUserProfileLoading } = useGetUserProfileQuery({
+    userProfileService,
+  });
+  const {
+    data: reportingHealth,
+    isLoading: isReportingHealthLoading,
+    isError: isReportingHealthError,
+  } = useGetReportingHealthQuery({ http });
+  const hasManageReportingPrivilege = useMemo(() => {
+    if (!capabilities) {
+      return false;
+    }
+    return capabilities.manageReporting.show === true;
+  }, [capabilities]);
+  const reportingPageLink = useMemo(
+    () => (
+      <EuiLink href={http.basePath.prepend(REPORTING_MANAGEMENT_SCHEDULES)}>
+        {i18n.REPORTING_PAGE_LINK_TEXT}
+      </EuiLink>
+    ),
+    [http.basePath]
+  );
+  const { mutateAsync: scheduleReport, isLoading: isScheduleExportLoading } = useScheduleReport({
+    http,
+  });
+  const { defaultTimezone } = useDefaultTimezone();
+  const now = useMemo(() => moment().tz(defaultTimezone), [defaultTimezone]);
+  const defaultStartDateValue = useMemo(() => now.toISOString(), [now]);
+  const schema = useMemo(
+    () =>
+      getScheduledReportFormSchema(
+        validateEmailAddresses,
+        availableReportTypes
+      ) as FormSchema<FormData>,
+    [availableReportTypes, validateEmailAddresses]
+  );
+  const recurring = true;
+  const startDate = defaultStartDateValue;
+  const timezone = defaultTimezone;
+  const { form } = useForm<FormData>({
+    defaultValue: scheduledReport,
+    options: { stripEmptyFields: true },
+    schema,
+    onSubmit: async (formData) => {
+      try {
+        const {
+          title,
+          reportTypeId,
+          recurringSchedule,
+          optimizedForPrinting,
+          sendByEmail,
+          emailRecipients,
+        } = formData;
+        // Remove start date since it's not supported for now
+        const { dtstart, ...rrule } = convertToRRule({
+          startDate: now,
+          timezone,
+          recurringSchedule,
+          includeTime: true,
+        });
+        await scheduleReport({
+          reportTypeId,
+          jobParams: getReportParams({
+            apiClient,
+            // The assertion at the top of the component ensures these are defined when scheduling
+            sharingData: sharingData!,
+            objectType: objectType!,
+            title,
+            reportTypeId,
+            ...(reportTypeId === 'printablePdfV2' ? { optimizedForPrinting } : {}),
+          }),
+          schedule: { rrule: rrule as Rrule },
+          notification: sendByEmail ? { email: { to: emailRecipients } } : undefined,
+        });
+        toasts.addSuccess({
+          title: i18n.SCHEDULED_REPORT_FORM_SUCCESS_TOAST_TITLE,
+          text: mountReactNode(
```

Comment:
`mountReactNode` should just be used internally in Core. Can you change this to `toMountPoint` from `@kbn/react-kibana-mount` ?